pyproject.toml

```diff
@@ -28,8 +28,8 @@ hydra-core = ">=1.0.4"
 matplotlib = ">=3.3.3"
 nflows = ">=0.14"
 numpy = ">=1.18.5"
-python = "^3.8"
-pytorch-lightning = ">=1.5.1"
+python = "^3.10"
```

Comment:
does this mean you want to drop support for python 3.8 and 3.9 ? I'm OK with this but I was curious of your rationale?